code/game/jobs/job/support.dm

```diff
@@ -244,6 +244,10 @@
 		H.equip_or_collect(new /obj/item/toy/crayon/rainbow(H), slot_in_backpack)
 		H.equip_or_collect(new /obj/item/weapon/storage/fancy/crayons(H), slot_in_backpack)
 		H.equip_or_collect(new /obj/item/weapon/reagent_containers/spray/waterflower(H), slot_in_backpack)
+		if(H.get_species() == "Machine")
```

Comment:
Should probably give this to all clowns, instead of just IPC clowns, that way no clown can escape the comic sans (without at least both surgery and genetics).